src/__tests__/label-helpers.js

```diff
@@ -0,0 +1,7 @@
+import {getRealLabels} from '../label-helpers'
+
+test('hidden inputs are not labelable', () => {
+  const element = document.createElement('input')
+  element.type = 'hidden'
+  expect(getRealLabels(element)).toEqual([])
```

Comment:
What was the result before your changes? 🤔

Reply:
Before the change it returns `null`:

<img width="564" alt="Screenshot 2020-11-03 at 17 44 01" src="https://user-images.githubusercontent.com/770763/98021534-516d1700-1dfc-11eb-898d-8492594d9446.png">

Reply:
I didn't see the PR and I have opened a PR on the same behaviour because I saw an issue about it.